src/config.py

```diff
@@ -66,3 +66,4 @@ def _convert_to_bool(x: str) -> bool:
 VESPA_CERT_LOCATION: str = os.getenv("VESPA_CERT_LOCATION", "")
 VESPA_KEY_LOCATION: str = os.getenv("VESPA_KEY_LOCATION", "")
 VESPA_NAMESPACE_PREFIX: str = os.getenv("VESPA_NAMESPACE_PREFIX", "navigator")
+DEV_MODE: bool = os.getenv("DEV_MODE", "False").lower() == "true"
```

Comment:
Think this should be `DEVELOPMENT_MODE` not `DEV_MODE`

Reply:
Good shout. Fixed